userland/examples/ble-env-sense/main.c

```diff
@@ -51,6 +51,7 @@ ble_advdata_manuf_data_t mandata = {
   .data.size   = sizeof(mdata)
 };
 
+__attribute__ ((const))
```

Comment:
Because it's behavior is const and we currently have a warning that detects cases where things are const functions but not marked as such